docs/background.md

```diff
@@ -2,22 +2,89 @@
+
+In the end, though women played a crucial role in the Easter Rising, their male comrades still saw them as different. Their role was a contentious one, varying from commander to commander: de Valera refused to have any women present in his garrison; James Connolly passed out revolvers to the women, yet refused to allow his daughter to fight at the GPO and stationed them where there was likely to be very little fighting.<sup>31</sup>   Many men even insisted that women run home before British assault began and the fall of the Republic, which most women refused.<sup>32</sup>  By the fall of the Republic a total of 80 women were taken into custody. Irish Rebels and British soldiers alike attempted to convince them to simply return to their homes, the women present had to vehemently insist on being arrested to the British officers.<sup>33</sup>  In all only five women, including Countess Markievicz, were sent to prison and the rest were cited as being ‘mislead’ by the men surrounding them and sent home.<sup>34</sup>
+
+**Religion in the Rising:**
+
+The role of religion in the Easter Rising was an undeniable factor, as an overwhelming number of those fighting were Catholics, in fact the only Protestant leader of the Rising was Countess Markievicz, who ended up converting to Catholicism.<sup>35</sup>  The rebellion even held claims of approval from the Vatican, as Count Plunkett told the Archbishop of Dublin that he himself had visited the Pope who blessed their desire to “act as Catholics.”<sup>36</sup>  This idea that they were acting for God reinforced the spirit of the Rebels, one even claimed if he died in the fighting without confession he “would go straight to heaven if I die here.”<sup>37</sup>  The event was steeped in the spirit of Catholicism, and there were many claims that divine intervention protected the Rebels from snipers as they kneeled to pray.<sup>38</sup>  Many prepared for the Rising itself with religious devotions throughout the weekend; the piety of the Rebels won over the skeptical public and changed public opinion of them positively.<sup>39</sup>  Few Protestants fought on the Rebel side of the Rising, and were often placed in more dangerous lookout spots.<sup>40</sup>
+
+---
 
-You have freedom to roam from the narrow topic of your project to explore how different authors/schools of thought have impacted scholarly approaches over time. However, make sure that in the end you clearly relate how this discussion relates to the subject of your project and/or your choices in methods or interpreative models.
+**Notes:**
```

Comment:
Great, and nice work with the footnotes. One thing, you need to add an extra blank space after **Notes** for the numbered list to work. Without the blank space, it reads it all as a single paragraph, starting with Notes and ending with the last numbered footnote.